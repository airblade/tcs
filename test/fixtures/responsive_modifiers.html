<div class="lg:grid-cols-4 grid sm:grid-cols-3 grid-cols-2">
</div>
