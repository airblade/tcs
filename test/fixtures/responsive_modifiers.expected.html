<div class="grid grid-cols-2 sm:grid-cols-3 lg:grid-cols-4">
</div>
